hamper/plugins/questions.py

```diff
@@ -102,11 +102,11 @@ def command(self, bot, comm, groups):
             r = random.random()
 
             flavor = [
-                (0.1, list(open('obliques.txt'))),
                 (0.02, ['Neither', 'None of them.']),
                 (0.02, ['Why not both?', 'Why not all of them?']),
                 (0.02, [u'¿Por qué no los dos?', u'¿Por qué no los todos?']),
             ]
+            flavor += [(.001, [x]) for x in list(open('oblique.txt'))]
```

Comment:
Close, but you'll need to do `(0.001, [x, x])`, since if there are 3 or more choices, hamper will try and take a second option.